### DISCUSSION TITLE
Set the Site Title for Exported empty.html

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6460] [Q&A] [answered]
In https://tiddlywiki.com there a button to export an empty clean Tiddlywiki.

**$:/editions/tw5.com/snippets/download-empty-button**

```
<$button class="tc-btn-download">
<$action-sendmessage $message="tm-download-file" $param="$:/editions/tw5.com/download-empty" filename="empty.html"/>
Download Empty {{$:/core/images/save-button}}
</$button>
```

The exported empty.html has a site title: **My TiddlyWiki**

Is it possible to set this through a variable? Is it possible to use a different site title for exported empty.html?
My use case is in Gatha, I need to be able to set different site title when I export empty.html for different plugin.

**Comment by saqimtiaz** [comment, marked as answer]
You are already injecting tiddlers into the new file you are creating for download, why not also inject `$:/SiteTitle` and set the site title as you want it?
